spec/lib/efile/md/md502_calculator_spec.rb

```diff
@@ -681,4 +653,287 @@
       expect(instance.lines[:MD502_LINE_D_AMOUNT_TOTAL].value).to eq 3200
     end
   end
+
+  describe "#calculate_line_1e" do
+    let(:filing_status) { "single" }
+
+    context 'when total interest is greater than $11,600' do
+      before do
+        intake.direct_file_data.fed_taxable_income = 11_599
+        intake.direct_file_data.fed_tax_exempt_interest = 2
+      end
+
+      it 'returns true' do
+        instance.calculate
+        expect(instance.lines[:MD502_LINE_1E].value).to be_truthy
+      end
+    end
+
+    context 'when total interest is less than or equal to $11,600' do
+      before do
+        intake.direct_file_data.fed_taxable_income = 11_599
+        intake.direct_file_data.fed_tax_exempt_interest = 1
+      end
+
+      it 'returns true' do
+        instance.calculate
+        expect(instance.lines[:MD502_LINE_1E].value).to be_falsey
+      end
+    end
+  end
+
+  describe "#gross_income_amount" do
+    let(:intake) { create(:state_file_md_intake, :head_of_household) } # needs to have fed_taxable_ssb element in order to set it; shelby_hoh has it
```

Comment:
[dust] is the comment here relevant? it doesn't look like shelby_hoh is being used

Reply:
it is being used! `:head_of_household` is a factory trait that attaches shelby_hoh as the fed xml